app/navigation/RegulatorsAndDocumentsNavigator.scala

```diff
@@ -39,10 +40,15 @@ class RegulatorsAndDocumentsNavigator @Inject()(implicit frontendAppConfig: Fron
           case _ => routes.SessionExpiredController.onPageLoad()
         }
 
-        case CharityCommissionRegistrationNumberPage => userAnswers: UserAnswers => userAnswers.get(CharityCommissionRegistrationNumberPage) match {
-          case Some(_) => routes.IndexController.onPageLoad()        // TODO modify once next page created
-          case _ => routes.SessionExpiredController.onPageLoad()     // TODO modify once next page created
-        }
+    case CharityCommissionRegistrationNumberPage => userAnswers: UserAnswers => userAnswers.get(CharityCommissionRegistrationNumberPage) match {
+      case Some(_) => regulatorAndDocumentInfoRoutes.CharityScottishRegulatorRegistrationNumberController.onPageLoad(NormalMode)
```

Comment:
could you please check exact match with Page Id, as we discussed in call, 
case Some(items)  if items.toSeq.contains(ScottishRegulatorRegistrationNumber) =>

Reply:
there is separate ticket for this change.so will cover this in that ticket